dsl/properties/scripts/DockerClient.groovy

```diff
@@ -701,6 +748,13 @@ public class DockerClient extends BaseClient {
         ComposeConfig composeConfig = composeFileReader.load(composeStream, workingDir, System.getenv())
         logger INFO, "composeContent: $composeConfig}"
         composeConfig
+
+    def getNetworkName(def serviceDetails){
+        formatName(getServiceParameter(serviceDetails, "networkName", serviceDetails.applicationName + "_default"))
```

Comment:
Do we need to default the network name if none is specified? What does default network that Docker swarm use if no network is specified? We should keep the same behavior.